### DISCUSSION TITLE
trame opens additional OpenGL window

### DISCUSSION THREAD

**Discussion by bigfooted** [Kitware/trame#39] [Q&A] [answered]
When I start trame I often get 2 windows, one in the firefox browser and one pop-up window showing the same image as in the browser. Often, the image in the popup-window is not updated after the initial creating (it becomes black). 
I get this for instance for this case: trame/examples/VTK/SimpleCone/LocalRendering.py
But not for a similar case: trame-tutorial/01_vtk/solution_cone.py
The title of the popup window is always "Visualization Toolkit - OpenGL"

I have the impression that the pop-up window is not supposed to happen, is this known behavior and can I suppress the creation of the popup window?

**Comment by jourdain** [comment, marked as answer]
What you are seeing is expected. The window is the VTK one where the rendering happen. You can programmatically disable it.

Using the LocalView will set a couple flags for you on the vtkRenderWindow, but if you call [Render()](https://github.com/Kitware/trame/blob/master/examples/VTK/SimpleCone/LocalRendering.py#L36) before those get set, the window will show up...

So just remove or comment that line. Also if VTK was built with EGL or OSMesa, you won't see such window either.

**Reply by bigfooted** [reply]
Great, thanks for your quick reply!

**Reply by jaumesagues** [reply]
I'm sorry but I do not understand well how I can avoid the pop-up window.
For instance, if I use the example 05_paraview/SimpleCone.py, if i use paraview.VtkLocalView I only see the cone in the pop-up window and the browser window is empty. However, if I use paraview.VtkRemoteView then I see the cone in the browser windos, and the pop-up window is empty. I would like to have only the browser window.

Thank you for your attention

**Reply by jourdain** [reply]
Here is what you can [call on a vtkRenderWindow](https://github.com/Kitware/trame-vtk/blob/master/examples/vue3/cone-local.py#L37) to disable the pop-up window.

**Reply by jourdain** [reply]
But a quick try indeed show an issue with the local view with ParaView. You may want to report a bug on trame-vtk pointing to that example.
